tasks/elasticsearch-RedHat.yml

```diff
@@ -19,7 +19,7 @@
   when: es_use_repository
   register: redhat_elasticsearch_install_from_repo
   notify: restart elasticsearch
-  until: '"failed" not in redhat_elasticsearch_install_from_repo'
+  until: redhat_elasticsearch_install_from_repo.failed != true
```

Comment:
I'm pretty sure this will fail when `failed` isn't defined (on fresh installs). So I think this line will need to be something like `"failed" in redhat_elasticsearch_install_from_repo and redhat_elasticsearch_install_from_repo.failed != true`. Lets see how the jenkins job goes though.

Reply:
Ahh yes I thought this was familiar, looks like someone else ran into the same problem. https://github.com/elastic/ansible-elasticsearch/pull/423/files#r169673578

Reply:
so this is a dup with same fix more or less. closing

Reply:
I didn't hear back from the previous person though, so if you have the time I would love to reopen this and get it fixed properly. The jenkins build failed as I expected:
```
13:20:12        TASK [elasticsearch : RedHat - Install Elasticsearch] **************************
13:20:20        fatal: [localhost]: FAILED! => {"msg": "The conditional check 'redhat_elasticsearch_install_from_repo.failed != true' failed. The error was: error while evaluating conditional (redhat_elasticsearch_install_from_repo.failed != true): 'dict object' has no attribute 'failed'"}
```

Reply:
ok, I will look into it..

Reply:
I've pushed a change, how do I trigger your CI to test it?

Reply:
It cancelled the old build and started again when you pushed. https://devops-ci.elastic.co/job/elastic+ansible-elasticsearch+pull-request/39/

Reply:
Woops, that retriggered when you reopened I think. Kicked off a new build: https://devops-ci.elastic.co/job/elastic+ansible-elasticsearch+pull-request/40/